feat: add Standard Notes Standalone Infrastructure setup instructions (#35)
standardnotes/docs: docs/self-hosting/configuration-options.md

---
slug: configuration-options/
id: configuration-options
title: Configuration options
sidebar_label: Configuration options
description: Manage options for your Standard Notes Standalone Infrastructure.
keywords:
  - standard notes
  - docs
  - notes app
  - end-to-end encryption
  - self-hosting
  - sync server
  - configuration options
image: /img/logo.png
hide_title: false
hide_table_of_contents: false
---

> ## Disclaimer

Reviewer: ```suggestion
## Note
```

Disclaimer makes it sound like something bad is coming up :)

> After changing any of the environment variables mentioned below you will have to restart the infrastructure in order for the changes to take place. You can do it by typing the following command:

Reviewer: ```suggestion
After changing any of the environment variables mentioned below you will have to restart the infrastructure in order for the changes to take effect. You can do so with the following command:
```
